grails-app/migrations/views/cycle-count-session.sql

```diff
@@ -8,13 +8,13 @@ SELECT
     product_availability.location_id                                                 as facility_id,
 
     -- Cycle count request ID
-    cycle_count_request_summary.cycle_count_request_id                               as cycle_count_request_id,
+    MAX(cycle_count_request_summary.cycle_count_request_id)                          as cycle_count_request_id,
 
     -- ABC Classification
-    -- FIXME Using a grouping operator due to a GROUP BY issue. This isn't the best approach sicne it'll return an
+    -- FIXME Using a grouping operator due to a GROUP BY issue. This isn't the best approach since it'll return an
     --  empty string or NULL value ahead of valid ABC classes (A, B, C, etc). But I don't have a better solution and
     --  this will likely work for 99% of cases.
-    MIN(COALESCE(inventory_level_summary.abc_class, product.abc_class))              as abc_class,
+    MIN(inventory_level_summary.abc_class)              as abc_class,
```

Comment:
Removed product.abc_class default to avoid JOIN (and because requirements aren't asking for this COALESCE). This has negligible impact on performance so we can add it back if necessary.

Reply:
Yeah, this was added before I joined product_classification. But I think you're right.